dotnet/src/SemanticKernel.Abstractions/AI/AudioToText/IAudioToTextService.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Services;
+
+namespace Microsoft.SemanticKernel.AudioToText;
+
+/// <summary>
+/// Interface for audio-to-text services.
+/// </summary>
+[Experimental("SKEXP0005")]
+public interface IAudioToTextService : IAIService
+{
+    /// <summary>
+    /// Get text content from audio binary data.
+    /// </summary>
+    /// <param name="audioData">Audio data.</param>
+    /// <param name="executionSettings">The AI execution settings (optional).</param>
+    /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>Text content from audio binary data.</returns>
+    Task<TextContent> GetTextContentAsync(
+        BinaryData audioData,
```

Comment:
Will we accept an audio data only as binary also in the other models, never as an URI?

Reply:
That's a good question, thanks. Most probably I will change it from `BinaryData` to `AudioContent` model, which is available in other PR for text-to-audio feature. `AudioContent` currently has `BinaryData`, but most probably will also contain `Uri` which will keep remote link to audio. Something similar should be done to `ImageContent` (there is already existing PR for that) for the ability to keep image content in different formats.